src/Engine/EmitMSIL/CodeGenIL.cs

```diff
@@ -159,7 +199,7 @@ private IEnumerable<MethodBase> FunctionLookup(IList args)
                     //}
                 }
             }
-            if (mi == null)
+            if (mi != null && mi.Count() > 0 == null)
```

Comment:
Do you mean `if( mi == null || (mi != null && mi.Count() > 0) )`?

Reply:
thanks, I think this was a copy paste error